### PR TITLE
Anastasiia/profile screen actions

### PR DESCRIPTION
**Pull request by spanastasia**

### CODE REVIEW

**Comment by oleksiikolomiietssnapp** [comment]
Nice job! 👍

**Review by oleksiikolomiietssnapp — APPROVED** [review]

File: INTITA/Modules/Profile/Cells/ProfileHeaderView.swift [hunk 1]
```diff
@@ -20,6 +23,7 @@ class ProfileHeaderView: UITableViewCell {
     @IBOutlet weak var specializationLabel: UILabel!
     
     weak var delegate: ProfileHeaderViewDelegate?
+    weak var dataSourse: ProfileHeaderViewDataSourse?
```
Comment (svitlana-korostelova) [inline INTITA/Modules/Profile/Cells/ProfileHeaderView.swift:26]: I guess there is no need for this row and protocol ProfileHeaderViewDataSourse in this PR, I haven't found callers. Or if it is a template for future tasks, let's write //TODO: inside.

File: INTITA/Modules/Profile/Cells/ProfileTableViewCell.swift [hunk 2]
```diff
@@ -7,23 +7,34 @@
...
+    func goToVC(number: Int)
+}
+
 class ProfileTableViewCell: UITableViewCell {
 
     @IBOutlet weak var container: UIView!
     @IBOutlet weak var button: UIButton!
     @IBOutlet weak var label: UILabel!
     
+    var row: Int?
```
Comment (svitlana-korostelova) [inline INTITA/Modules/Profile/Cells/ProfileTableViewCell.swift:20]: Is there a necessity to make it optional? Row could be absent?

File: INTITA/Modules/Profile/Cells/ProfileHeaderView.swift [hunk 3]
```diff
@@ -8,10 +8,13 @@
...
 protocol ProfileHeaderViewDelegate: AnyObject {
-    func editImage()
     func avatarTapped()
 }
 
+protocol ProfileHeaderViewDataSourse: AnyObject {
+    func editImage()
+}
+
 class ProfileHeaderView: UITableViewCell {
```
Comment (svitlana-korostelova) [inline INTITA/Modules/Profile/Cells/ProfileHeaderView.swift:18]: Please, name class with "...Cell" at the end

File: INTITA/Modules/Profile/Cells/ProfileHeaderView.swift (hunk 1 above)
Comment (spanastasia) [reply INTITA/Modules/Profile/Cells/ProfileHeaderView.swift:26]: I forgot to remove it. Thanks

File: INTITA/Modules/Profile/Cells/ProfileTableViewCell.swift (hunk 2 above)
Comment (spanastasia) [reply INTITA/Modules/Profile/Cells/ProfileTableViewCell.swift:20]: If make it non-optional it is necessary to provide a constructor. I want to use tableView.dequeReusableCell() that's why row is optional.

File: INTITA/Modules/Profile/Cells/ProfileHeaderView.swift (hunk 3 above)
Comment (spanastasia) [reply INTITA/Modules/Profile/Cells/ProfileHeaderView.swift:18]: done

**Pull request merged**